infrastructure/development/tasks.tf

```diff
@@ -0,0 +1,125 @@
+### DEVELOPER PORTAL TASKS
+
+resource "azurerm_app_service_plan" "azurerm_app_service_plan_portal" {
+  name                = "${local.azurerm_app_service_plan_portal_name}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+
+  sku {
+    tier = "Standard"
+
+    # see https://azure.microsoft.com/en-en/pricing/details/app-service/
+    size = "S1"
+  }
+}
+
+resource "random_string" "cookie_key" {
+  length = 32
+}
+
+resource "random_string" "cookie_iv" {
+  length = 12
+}
+
+resource "azurerm_app_service" "azurerm_app_service_portal" {
+  name                = "${local.azurerm_app_service_portal_name}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+  app_service_plan_id = "${azurerm_app_service_plan.azurerm_app_service_plan_portal.id}"
+
+  site_config {
+    always_on = true
+  }
+
+  # Go to https://github.com/teamdigitale/digital-citizenship-onboarding
+  # to see how to fill these values
+  app_settings {
+    POLICY_NAME                  = "${var.azurerm_adb2c_policy}"
+    WEBSITE_NODE_DEFAULT_VERSION = "6.5.0"
+    COOKIE_KEY                   = "${random_string.cookie_key.result}"
+    COOKIE_IV                    = "${random_string.cookie_iv.result}"
+    LOG_LEVEL                    = "info"
+    ARM_RESOURCE_GROUP           = "${azurerm_resource_group.azurerm_resource_group.name}"
+    ARM_APIM                     = "${local.azurerm_apim_name}"
+    APIM_PRODUCT_NAME            = "starter"
+    APIM_USER_GROUPS             = "ApiLimitedMessageWrite,ApiInfoRead,ApiMessageRead,ApiLimitedProfileRead"
+    ADMIN_API_URL                = "https://${local.azurerm_apim_name}.azure-api.net/"
+    POST_LOGIN_URL               = "${var.app_service_portal_post_login_url}"
+    POST_LOGOUT_URL              = "${var.app_service_portal_post_logout_url}"
+    REPLY_URL                    = "https://${local.azurerm_app_service_portal_name}.azurewebsites.net/auth/openid/return"
+
+    ARM_SUBSCRIPTION_ID = "${data.azurerm_client_config.current.subscription_id}"
+    TENANT_ID           = "${var.ADB2C_TENANT_ID}"
+    CLIENT_ID           = "${var.DEV_PORTAL_EXT_CLIENT_ID}"
+    CLIENT_SECRET       = "${var.DEV_PORTAL_EXT_CLIENT_SECRET}"
+
+    # Prevent Terraform to override these values
+    APPINSIGHTS_INSTRUMENTATIONKEY = ""
```

Comment:
@gunzip do you remember why we needed this?

Reply:
because if you setup these later (ie. through the provisioner) another run of terraform will overwrite those values.